go.mod

```diff
@@ -76,31 +76,31 @@ require (
 	github.com/tonistiigi/units v0.0.0-20180711220420-6950e57a87ea
 	github.com/tonistiigi/vt100 v0.0.0-20240514184818-90bafcd6abab
 	github.com/urfave/cli v1.22.16
-	github.com/vishvananda/netlink v1.2.1-beta.2
+	github.com/vishvananda/netlink v1.3.0
 	go.etcd.io/bbolt v1.3.11
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.46.1
 	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.46.1
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.1
-	go.opentelemetry.io/otel v1.21.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.53.0
+	go.opentelemetry.io/otel v1.28.0
```

Comment:
Do we need to bump semconv to 1.26.0? https://github.com/moby/buildkit/tree/master/vendor/go.opentelemetry.io/otel/semconv

Seems we are inconsistent in our code base:
* https://github.com/moby/buildkit/blob/8cbb9170e015b5174f11d29310da2e8c9dd60d9e/util/tracing/tracing.go#L14
* https://github.com/moby/buildkit/blob/8cbb9170e015b5174f11d29310da2e8c9dd60d9e/util/tracing/detect/resource.go#L11

Reply:
I see containerd is still on 1.21.0: https://github.com/containerd/containerd/blob/207ad711eabd375a01713109a8a197d197ff6542/pkg/tracing/tracing.go#L28

Reply:
Ah forgot we had this one pending https://github.com/moby/buildkit/pull/5460